databuilder/loader/file_system_neo4j_csv_loader.py

```diff
@@ -37,7 +37,7 @@ class FsNeo4jCSVLoader(Loader):
 
     _DEFAULT_CONFIG = ConfigFactory.from_dict({
         SHOULD_DELETE_CREATED_DIR: True,
-        FORCE_CREATE_DIR: False
+        FORCE_CREATE_DIR: True
```

Comment:
any reason with this change?

Reply:
left that in by mistake. I'll remove it.